### PR TITLE
Alot of little adds and possible fixes...

### PR DESCRIPTION
**Pull request by GoldenMariaNova**
**First** of all, the new hair texture for the **OOT** original and compatible models, ported and adjusted from **MM** : 
**_Child :_**
367557754-e4d18541-18aa-4366-928f-dcb8b8565df0
**_Adult :_**
367557786-95b6f839-55ae-45ee-b8c0-5a800f738275

**Second**, I've ported a certain set of codes from **Child Quest Classic**, allowing child **Link** to buy tunics in shops, just like his adult form when selecting "**Unlock Tunics**" : 
**_Goron Tunic :_**
367557918-ebe092bf-bc7b-4a7d-8313-99a4f6cb0ad9
**_Zora Tunic :_**
367557956-267c49d8-a5f9-4acf-b6f3-456bcd29286b

The text was also re-adjusted and fixed, as some text overflow would happen in both "_you just bought_" messages. Everything tested, no bugs, not even in adult form.

**Third**, the "**Instant Claim Check**" option replaces some dialogues now, has the many things the **Biggoron** would says during his respective sidequest, was still unchanged, making no sense at all, like "_waiting a few days_", for example. Now, it's contextualized correctly.

I hope these changes will be... satisfactory. Like I said earlier, it's all tested.

### CODE REVIEW

**Comment by GoldenMariaNova** [comment]
This **Pull Request** is a **_MUCH_** more cleaned, "_less useless commits_" version of : https://github.com/Admentus64/Patcher64Plus-Tool/pull/150.

My fork from **Patcher64+** is now bug-free. Sorry for all the trouble.

**Comment by Admentus64** [comment]
Alright. I review it some time later.

**Comment by GoldenMariaNova** [comment]
A little update, some **Missing Link** _HUD/Hearts_ were either badly compressed or had stray pixels.

It's fixed now.

**Comment by GoldenMariaNova** [comment]
It should be enough for now. Everything is verified as usual.

**Comment by Admentus64** [comment]
Reviewed. Looks fine. Only got a single comment. Please check.

File: Files/Scripts/Options/Ocarina of Time.psm1 [hunk 1]
```diff
@@ -2566,7 +2571,7 @@ function CreatePresets() {
         $FemaleModels  = CreateReduxButton -Width 150 -Text "Female Link"
 
         $VanillaModels.Add_Click( { $Redux.Graphics.ChildModels.SelectedIndex =                  $Redux.Graphics.AdultModels.SelectedIndex = 0                  ; BoxUncheck $Redux.Text.FemalePronouns; $Redux.Sounds.ChildVoices.SelectedIndex = $Redux.Sounds.AdultVoices.SelectedIndex = 0       } )
-        $MajoraModels.Add_Click(  { $Redux.Graphics.ChildModels.Text          = "Majora's Mask"; $Redux.Graphics.AdultModels.Text          = "Link Baldrics"    ; BoxUncheck $Redux.Text.FemalePronouns; $Redux.Sounds.ChildVoices.SelectedIndex = $Redux.Sounds.AdultVoices.SelectedIndex = 0       } )
+        $MajoraModels.Add_Click(  { $Redux.Graphics.ChildModels.Text          = "Majora's Mask"; $Redux.Graphics.AdultModels.Text          = "MMLink"           ; BoxUncheck $Redux.Text.FemalePronouns; $Redux.Sounds.ChildVoices.SelectedIndex = $Redux.Sounds.AdultVoices.SelectedIndex = 0       } )
```
[the remark this answers is not in the document]
Comment (GoldenMariaNova) [reply Files/Scripts/Options/Ocarina of Time.psm1]: It have been changed to default **OOT** Adult Link for the time being.

**Comment by GoldenMariaNova** [comment]
OK, an Update on something that should have been done long ago.

First, here's a reference : https://tcrf.net/The_Legend_of_Zelda:_Ocarina_of_Time/Version_Differences

Text sample : 
**Dampé's Grave Gerudo Symbol**
_Despite being removed in the Gateway build, the old Gerudo symbol texture in the first and last rooms of Dampé's Grave is present again in all of the GameCube builds. This was caught and corrected for all later emulated releases with a new texture._

It's also an remnant of **OOT** older dev builds : https://youtu.be/HyUDlTb_8jI?t=2462 , for reference. It was for some kind of maze that got repurposed as the **Dampe**'s race later, thus this texture got actually no use at all in the final version.

So, with all that, an combo box and new options have been added. Either just remove that texture, or in addition, change the symbols aswell.

Now, this is like **Nintendo** should have done years ago.

**Comment by GoldenMariaNova** [comment]
By the way, please merge my fork, since I uploaded and merged **GhostlyDark** recent changes to my repository.

**Comment by Admentus64** [comment]
I have to review it again due to all changes.

I seen some new issues already.

The new Gerudo censor shouldn't be a new option.

I am also removing model presets since we no longer host models ourselves. So that change has to be removed.

**Comment by GoldenMariaNova** [comment]
> snips

Please understand I did it this way, because someone should be able to remove **Dampe**'s **Gerudo** symbol while keeping the original designs. 

If someone choose to censor them instead, they will still benefit from the fix, in addition of the now changed aforementioned symbols. Like I said, it shouldn't be there in the first place. 

I tried to separate them, but it didn't work. It's the only way it even works in the first place.

So fusing these two options is the best course of action, believe me.

**Comment by GoldenMariaNova** [comment]
> snips

If anything, now the text is more specific on what it does now.

**Comment by Admentus64** [comment]
It's fine. Never mind my comments. Just thinking too hard on it.

I merge it soon, but currently I am very occupied.

**Comment by GoldenMariaNova** [comment]
> snips

Thank you. And I wait.

**Comment by Admentus64** [comment]
Merged

**Pull request merged**